### ISSUE TITLE
Android interactive login - null pointer exception

### ISSUE THREAD

**Issue by tricertc** [recognizegroup/capacitor-plugin-msauth#2]
Thank you for your work on this plugin!  I'm trying to implement it, but am hitting a null pointer exception on the android side. It's only happening during interactive login, silent login works fine.

I think I've narrowed it down to lines #141-142 in MsAuthPlugin.java.

https://github.com/recognizegroup/capacitor-plugin-msauth/blob/main/android/src/main/java/nl/recognize/msauthplugin/MsAuthPlugin.java#L141-L142

```java
tokenResult.setAccessToken(tokenResult.getAccessToken());
tokenResult.setScopes(tokenResult.getScopes());
```

Should the access token and scopes properties be set to the values received in the authenticationResult instead?

```java
tokenResult.setAccessToken(authenticationResult.getAccessToken());
tokenResult.setScopes(authenticationResult.getScope());
```

**Comment by bartwesselink** [comment]
Hi @tricertc! Thank you for using the plugin. Do you have the full exception and stack trace for me?

**Comment by tricertc** [comment]
Here's what I was able to capture in the logs. 

```
2022-01-28 10:32:12.541 3197-3197/? E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.demo.app, PID: 3197
    java.lang.NullPointerException
        at java.util.Objects.requireNonNull(Objects.java:220)
        at java.util.Arrays$ArrayList.<init>(Arrays.java:3738)
        at java.util.Arrays.asList(Arrays.java:3725)
        at nl.recognize.msauthplugin.MsAuthPlugin.lambda$login$0(MsAuthPlugin.java:63)
        at nl.recognize.msauthplugin.-$$Lambda$MsAuthPlugin$p9PTtsguqM-uYaAnpO2MpdHcRQI.tokenReceived(Unknown Source:2)
        at nl.recognize.msauthplugin.MsAuthPlugin$2.onSuccess(MsAuthPlugin.java:144)
        at com.microsoft.identity.client.PublicClientApplication.postAuthResult(PublicClientApplication.java:2002)
        at com.microsoft.identity.client.SingleAccountPublicClientApplication$3.onTaskCompleted(SingleAccountPublicClientApplication.java:345)
        at com.microsoft.identity.client.SingleAccountPublicClientApplication$3.onTaskCompleted(SingleAccountPublicClientApplication.java:336)
        at com.microsoft.identity.common.internal.controllers.CommandDispatcher.commandCallbackOnTaskCompleted(CommandDispatcher.java:448)
        at com.microsoft.identity.common.internal.controllers.CommandDispatcher.access$900(CommandDispatcher.java:83)
        at com.microsoft.identity.common.internal.controllers.CommandDispatcher$3.run(CommandDispatcher.java:427)
        at android.os.Handler.handleCallback(Handler.java:938)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:223)
        at android.app.ActivityThread.main(ActivityThread.java:7656)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
```

**Comment by bartwesselink** [comment]
Fixed in v2.0.1
